python/paddle/fluid/tests/unittests/mkldnn/test_matmul_v2_mkldnn_op.py

```diff
@@ -249,7 +247,8 @@ def set_dtype_attr(self):
             self.attrs['mkldnn_data_type'] = "bfloat16"
 
         def test_check_output(self):
-            self.check_output_with_place(core.CPUPlace())
+            if core.supports_bfloat16():
+                self.check_output_with_place(core.CPUPlace())
```

Comment:
Could you make some notification visible to the outside (CI log) if core does not support bf16. Otherwise we will not know if test was actually run.

Reply:
I'll add that